share/spice/quandl/fundamentals/quandl_fundamentals.css

```diff
@@ -0,0 +1,74 @@
+/*
+    Document   : 
+    Created on : 2015-02-12 08:21 PM
+    Author     : brianrisk
+    Description: styles based on the 'Stocks' spice for visual consistency
+*/
+
+#quandl h1.quandl-header {
```

Comment:
please use `zci--quandl_fundamentals` instead of `#quandl` for namespacing the CSS. That's the canonical method. Then you can also remove the `<div id="quandl">`